Compilation: Failed
Output:
fn foo() { 42 };
   ^
3:4: Duplicate function
